meta-mender-tegra/recipes-mender/mender-client/files/mender

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+echo "Starting mender wrapper"
+
+echo $@ | grep 'install\|-install' > /dev/null
```

Comment:
@jajoosiddhant include spaces in the name here to prevent match with argument content (like filenames)